mk/spksrc.wheel.mk

```diff
@@ -63,9 +63,9 @@ build_wheel_target: $(PRE_WHEEL_TARGET)
 		if [ ! -z "$(CROSS_COMPILE_WHEELS)" ] ; then \
 			$(MSG) "Force cross-compile" ; \
 			if [ -z "$(CROSSENV)" ]; then \
-				$(RUN) _PYTHON_HOST_PLATFORM="$(TC_TARGET)" CFLAGS="$(CFLAGS) -I$(STAGING_INSTALL_PREFIX)/$(PYTHON_INC_DIR) $(WHEELS_CFLAGS)" LDFLAGS="$(LDFLAGS) $(WHEELS_LDFLAGS)" $(PIP_WHEEL) ; \
```

Comment:
It turnsout that using `-I` wasn't appreciated in some cases (more specifically `gevent`) where it was failing, indicating that it should go through `CPPFLAGS` instead.